addons/spreadsheet/static/tests/pivots/model/pivot_table_function.test.js

```diff
@@ -281,13 +281,13 @@ test("PIVOT with multiple row groups", async function () {
         [null,          "xphone",       "xpad",         "Total"],
         [null,         "Probability",  "Probability",  "Probability"],
         [1,             "",             11,             11],
-        [2,             "",             11,             11],
+        ["None",        "",             11,             11],
         [2,             "",             15,             15],
-        [4,             "",             15,             15],
+        ["None",        "",             15,             15],
         [12,            10,             "",             10],
-        [1,             10,             "",             10],
+        ["None",        10,             "",             10],
         [17,            "",             95,             95],
-        [3,             "",             95,             95],
+        ["None",        "",             95,             95],
```

Comment:
Maybe add display names to the demo data, so we don't end up with "None" everywhere here, which doesn't test much